### PR TITLE
update the documentation a bit

### PR DESCRIPTION
**Pull request by hboetes**
- Since dependencies are important to both people who want to run from git and packagers I created DEPENDENCIES.md and I move various dependencies mentions in other files to this one, replacing them with links to the new file 
- Move licence next to legal stuff
- Improve the descriptions a bit
- Improve wording a bit
- Improve the layout a bit

### CODE REVIEW

**Comment by hboetes** [comment]
Some check were not successful? What did I do? :-D

File: README.md [hunk 1]
```diff
@@ -32,24 +32,25 @@ $ sudo apt install nicotine
...
 
 ## Windows
-- [Nicotine+ Installer](https://github.com/Nicotine-Plus/nicotine-plus/releases/download/2.0.0/Nicotine+-2.0.0.exe)  
+- [Nicotine+ Installer](https://github.com/Nicotine-Plus/nicotine-plus/releases/download/2.0.0/Nicotine+-2.0.0.exe)
 
-# License
-
-Nicotine+ is released under the terms of the [GNU Public License v3](https://www.gnu.org/licenses/gpl-3.0-standalone.html) or later.
+## Building from git
+Building from git is described in [RUNFROMGIT.md](doc/RUNFROMGIT.md)
```
Comment (mathiascode) [inline README.md:38]: Add a sentence or two about this only being encouraged for testers, e.g. https://github.com/Nicotine-Plus/nicotine-plus/pull/392/files

File: doc/RUNFROMGIT.md [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
-* In the git root folder, run `python nicotine`.
-
-## Windows
-
-See [PACKAGING.md](doc/PACKAGING.md#windows)
+## Installing the dependencies
+To run nicotine from git you first have to install all required dependencies. You may also want to install the optional dependencies.
+See [DEPENDENCIES.md](doc/DEPENDENCIES.md)
+
+## Clone the repository and run nicotine
```
Comment (mathiascode) [inline doc/RUNFROMGIT.md:7]:
```suggestion
## Clone the repository and run Nicotine+
```

File: doc/RUNFROMGIT.md [hunk 3]
```diff
@@ -1,31 +1,30 @@
...
+This is great if you're the only nicotine user on your system and you want to test the latest version.
```
Comment (mathiascode) [inline doc/RUNFROMGIT.md:8]:
```suggestion
This is great if you're the only Nicotine+ user on your system and you want to test the latest version.
```

File: doc/RUNFROMGIT.md [hunk 4]
```diff
@@ -1,31 +1,30 @@
...
+```
+$ git clone https://github.com/Nicotine-Plus/nicotine-plus.git
+$ cd nicotine-plus
+$ python3 nicotine
+```
+
+## Updating your cloned nicotine-plus
```
Comment (mathiascode) [inline doc/RUNFROMGIT.md:15]:
```suggestion
## Updating your cloned Nicotine+
```

File: doc/RUNFROMGIT.md [hunk 5]
```diff
@@ -1,31 +1,30 @@
...
+If you want to update to the latest version of nicotine-plus proceed like this:
```
Comment (mathiascode) [inline doc/RUNFROMGIT.md:16]:
```suggestion
If you want to update to the latest version of Nicotine+, proceed like this:
```

File: doc/RUNFROMGIT.md [hunk 6]
```diff
@@ -1,31 +1,30 @@
...
+```
+$ cd nicotine-plus
+$ git pull
+$ python3 nicotine
+```
+
+## Installing nicotine from the git repository
```
Comment (mathiascode) [inline doc/RUNFROMGIT.md:23]:
```suggestion
## Installing Nicotine+ from the git repository
```

File: doc/RUNFROMGIT.md [hunk 7]
```diff
@@ -1,31 +1,30 @@
...
+This is useful if you want to share your nicotine-plus installation with other users.
```
Comment (mathiascode) [inline doc/RUNFROMGIT.md:24]:
```suggestion
This is useful if you want to share your Nicotine+ installation with other users.
```

File: doc/RUNFROMGIT.md [hunk 8]
```diff
@@ -1,31 +1,30 @@
...
+```
+$ cd nicotine
```
Comment (mathiascode) [inline doc/RUNFROMGIT.md:26]:
```suggestion
$ cd nicotine-plus
```

File: doc/RUNFROMGIT.md [hunk 9]
```diff
@@ -1,31 +1,30 @@
...
-
-* Check that the Python version you are using is 3.5 or newer via `python -V`.
-
-* In the git root folder, run `python nicotine`.
-
-## Windows
-
-See [PACKAGING.md](doc/PACKAGING.md#windows)
+## Installing the dependencies
+To run nicotine from git you first have to install all required dependencies. You may also want to install the optional dependencies.
```
Comment (mathiascode) [inline doc/RUNFROMGIT.md:4]:
```suggestion
To run Nicotine+ from git, you first have to install all required dependencies. You may also want to install the optional dependencies.
```

**Comment by mathiascode** [comment]
Nothing wrong with your changes. I think GitHub Actions is just having a bad day.

File: README.md (hunk 1 above)
Comment (hboetes) [reply README.md:38]: Yes, good idea. I've added that to the RUNFROMGIT.md. Now I have to figure out if the change made it into the pull request.

Comment (hboetes) [reply README.md:38]: Please check https://github.com/hboetes/nicotine-plus/blob/master/README.md to get an overview on the whole README. I put the build from git on the logical spot, as the last of the installation options, and the licence next to other legal stuff.
I hope that satisfies your request.

**Pull request closed without merging**